cmd/config-seed/main.go

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright 2018 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/edgexfoundry/edgex-go/internal"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
+	"github.com/edgexfoundry/edgex-go/internal/seed/config"
+	"github.com/edgexfoundry/edgex-go/pkg/clients/logging"
+	"sync"
+)
+
+var bootTimeout int = 30000 //Once we start the V2 configuration rework, this will be config driven
+
+func main() {
+	var useProfile string
+	var dirCmd string
+	var dirProperties string
+
+	flag.StringVar(&useProfile, "profile", "", "Specify a profile other than default.")
+	flag.StringVar(&useProfile, "p", "", "Specify a profile other than default.")
+	flag.StringVar(&dirProperties, "props", "./res/properties", "Specify alternate properties location as absolute path")
+	flag.StringVar(&dirProperties, "r", "./res/properties", "Specify alternate properties location as absolute path")
+	flag.StringVar(&dirCmd, "cmd", "../cmd", "Specify alternate cmd location as absolute path")
+	flag.StringVar(&dirCmd, "c", "../cmd", "Specify alternate cmd location as absolute path")
+
+	flag.Usage = usage.HelpCallback
```

Comment:
`usage.HelpCallback` it's not valid in this case

Reply:
@jduranf Good catch. Config-seed specific support has been added.